templates/download/intel-nuc.html

```diff
@@ -1,135 +1,196 @@
-            <li class="p-list__item is-ticked">An Ubuntu Core image</li>
-          </ul>
-        </div>
-      </div>
+    <div class="col-6 col-medium-3">
+      <p>There are two install options for the Intel NUC: <a href="/download/intel-nuc">Ubuntu Core</a> or <a href="/download/intel-nuc-desktop">Ubuntu Desktop</a>  This page is for Ubuntu Core.</p>
+      <p>We will walk you through the steps of flashing Ubuntu Core on an Intel NUC. At the end of this process, you will have a board ready for production or testing snaps.</p>
+      <hr class="p-rule" />
+      <p class="p-heading--5">Minimum requirements</p>
+      <ul class="p-list--divided">
+        <li class="p-list__item is-ticked">An Ubuntu SSO account with an SSH key</li>
+        <li class="p-list__item is-ticked">An Intel NUC with BIOS updated to the latest version (<a href="https://www.intel.com/content/www/us/en/support/articles/000033300/intel-nuc.html">update instructions</a>)</li>
+        <li class="p-list__item is-ticked">2 USB 2.0 or 3.0 flash drives (2GB minimum)</li>
+        <li class="p-list__item is-ticked">A monitor with an HDMI interface</li>
+        <li class="p-list__item is-ticked">A Mini HDMI to HDMI cable</li>
+        <li class="p-list__item is-ticked">A USB keyboard and a mouse</li>
+        <li class="p-list__item is-ticked">A monitor with VGA or HDMI interface</li>
+        <li class="p-list__item is-ticked">A VGA or HDMI cable</li>
+        <li class="p-list__item is-ticked">A network connection with Internet access</li>
+        <li class="p-list__item is-ticked">An Ubuntu Desktop {{ releases.lts.full_version }} <abbr title="Long-term support">LTS</abbr> image</li>
+        <li class="p-list__item is-ticked">An Ubuntu Core image</li>
+      </ul>
     </div>
   </div>
 </section>
 
-<section class="p-strip is-deep is-bordered">
+<section class="p-section">
   <div class="u-fixed-width">
-    <h2>Installation instructions</h2>
     <ol class="p-stepped-list--detailed">
-      {% include "download/iot/_setup-ubuntu-sso.html" %}
       <li class="p-stepped-list__item">
-        <h3 class="p-stepped-list__title">
-
+        <hr class="p-rule u-hide--large" />
+        <h2 class="p-stepped-list__title">
+          Set up an Ubuntu SSO account
+        </h2>
+        <div class="p-stepped-list__content">
+          <p>An Ubuntu SSO account is required to create the first user on an Ubuntu Core installation.</p>
+          <ol class="p-list--divided">
+            <li class="p-list__item">Start by creating an <a href="https://login.ubuntu.com/">Ubuntu SSO account</a>.</li>
+            <li class="p-list__item">Import an SSH Key into your <a href="https://login.ubuntu.com/ssh-keys">Ubuntu SSO account</a>. (<a href="https://help.ubuntu.com/community/SSH/OpenSSH/Keys">instructions</a>)</li>
+          </ol>
+        </div>
+      </li>
```

Comment:
I thought we agreed to use existing partials. Why repeat this code if it's word for word exactly the same?